test/__snapshots__/test/test.ts.md

```diff
@@ -279,6 +279,153 @@ Generated by [AVA](https://ava.li).
     }␊
     `
 
+## arrayAdditionalItems.js
+
+> Snapshot 1
+
+    `/* tslint:disable */␊
+    /**␊
+     * This file was automatically generated by json-schema-to-typescript.␊
+     * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,␊
+     * and run json-schema-to-typescript to regenerate this file.␊
+     */␊
+    ␊
+    export interface ArrayAdditionalItems {␊
+      namedEums?: {␊
+        additionalItemsAny?: [NamedEnum2, ...any[]];␊
```

Comment:
I wonder if this should be `...unknown[]` instead. Or maybe we should add a `--strictNoAny` flag in the future, so that all generated `any`s (not just this one) are `unknown` instead.

Reply:
I'd probably lean toward the later solution I think, just so that the output is consistent across the package (it seems weird to introduce a single `unknown` generation point).